ano/environment.py

```diff
@@ -215,7 +217,8 @@ def find_arduino_user_dir(self, key, dirname_parts, items=None, human_name=None,
         return self.find_dir(key, items, self.arduino_user_places(dirname_parts), human_name, multi=multi, optional=optional)
 
     def find_arduino_file(self, key, dirname_parts, items=None, human_name=None, multi=False):
-        return self.find_file(key, items, self.arduino_dist_places(dirname_parts), human_name, multi=multi)
+        places = self.arduino_user_places(dirname_parts) + (self.arduino_dist_places(dirname_parts))
+        return self.find_file(key, items, places, human_name, multi=multi)
```

Comment:
This is the another part that actually matters. Basically just search more locations